db/schema.rb

```diff
@@ -33,9 +34,9 @@
 
   create_table "certifications", force: :cascade do |t|
     t.integer  "user_id"
-    t.string   "name"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "created_at",          null: false
+    t.datetime "updated_at",          null: false
+    t.integer  "training_session_id"
```

Comment:
You may have to reset your database and then run `rake db:schema:dump` because this shouldn't be here